### PR TITLE
docs: add structured vs raw content explanation

### PR DESCRIPTION
**Pull request by NickIliev**
Related to https://kinvey.atlassian.net/browse/FID-4233

### CODE REVIEW

**Review by rosen-vladimirov — APPROVED** [review]

File: compose-api-requests/api-requests-composer.md [hunk 1]
```diff
@@ -48,7 +48,14 @@ Once the API request executes, you can inspect its HTTP Response in the Composer
 
 1. Capture a session (or open a session from the saved save **Sessions** tree) and use the **Edit in Composer** option from the context menu.
 
-As a result, the session loads as an HTTP Request in the Fiddler's Composer, where you can make atomic modifications to its structure.
+As a result, the session loads as an HTTP Request in the Fiddler's Composer, where you can make atomic modifications to its structure. The **Composer** has a flexible user interface that allows you to switch between **Structured** and **Raw** content (refer to the screenshots below) and between **Table** or **Bulk** style inspectors.
```
Comment (yordan-mitev) [inline compose-api-requests/api-requests-composer.md:51]:
```suggestion
As a result, the session loads as an HTTP request in Fiddler's **Composer**, where you can make atomic modifications to its structure. The **Composer** has a flexible user interface that allows you to switch between **Structured** and **Raw** content (refer to the screenshots below) and between **Table** or **Bulk** style inspectors.
```

File: compose-api-requests/api-requests-composer.md [hunk 2]
```diff
@@ -48,7 +48,14 @@ Once the API request executes, you can inspect its HTTP Response in the Composer
...
+
+
+Example for using the **Structured** mode
```
Comment (yordan-mitev) [inline compose-api-requests/api-requests-composer.md:54]: I suggest using a bullet list for the examples.
```suggestion
* Using the **Structured** mode
```

File: compose-api-requests/api-requests-composer.md [hunk 3]
```diff
@@ -48,7 +48,14 @@ Once the API request executes, you can inspect its HTTP Response in the Composer
...
+    ![structured mode in Composer](../images/composer/composer-structured.png)
+
+Example for using the **Raw** mode
```
Comment (yordan-mitev) [inline compose-api-requests/api-requests-composer.md:57]:
```suggestion
* Using the **Raw** mode
```

**Review by yordan-mitev — APPROVED** [review]
Looks great, I'm adding just a few small suggestions.

**Pull request merged**